advance_ai_agents/candidate analyser/hiring_prompts.yaml

```
@@ -0,0 +1,155 @@
+description_for_multi_candidates: |
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)